src/SFA.DAS.EmployerIncentives.Domain/Accounts/LegalEntityVrfCaseStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace SFA.DAS.EmployerIncentives.Domain.Accounts
+{
+    public static class LegalEntityVrfCaseStatus
```

Comment:
Do we need the LegalEntity bit in the name?

Reply:
It's the VRF case status for a given legal entity so thought that naming would be an indication of scope